### PR TITLE
Configuration File Documentation

### PR DESCRIPTION
**Pull request by LogoiLab**
I have started work on the config file information page for the online docs. I still need the path for both macOS and Windows.

### CODE REVIEW

File: docs/docs/config-file.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+| Linux | `~/.local/share/xemu/xemu/xemu.ini` |
+| macOS |  |
+| Windows |  |
+
+
+## Default `xemu.ini` File
+
+The default configuration file looks like this:
+
+```ini
```
Comment (mborgerson) [inline docs/docs/config-file.md:16]: Prefer to exclude. Section is not necessary, and subject to change. Default config file will be created anyway for people who want to look at it.

File: docs/docs/config-file.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+## Paths
+
+The default xemu configuration file can be found at the following locations after first run:
+
+| OS | Path |
+|----|------|
+| Linux | `~/.local/share/xemu/xemu/xemu.ini` |
+| macOS |  |
```
Comment (mborgerson) [inline docs/docs/config-file.md:8]: macOS: /Users/username/Library/Application\ Support/xemu/xemu/xemu.ini
Windows: C:\Users\username\AppData\Roaming\xemu\xemu\xemu.ini

Should note that xemu.ini can be created in xemu executable directory, or explicitly given via `-config_path` command line argument.

**Comment by MasonT8198** [comment]
@mborgerson may not agree but I feel that it could fit well into the FAQ.md question, I don’t feel it needs a page all to itself

**Comment by LogoiLab** [comment]
Its up to you guys where you want to put it. I feel like users may hunt for it too much if it's in CLI. Perhaps an advanced page for all of the advanced items?

**Comment by MasonT8198** [comment]
Awaiting Author Response/Requested Fixes

**Comment by MasonT8198** [comment]
This needs pretty much completely redone after the latest changes to the configuration file format :)

consider closing unless the author decides to commit to the upkeep

**Pull request closed without merging**